sdk/kfp-tekton-limitation.md

```diff
@@ -0,0 +1,27 @@
+
+# Limitations
+
+- [Overview](#overview)
+    -[Affinity, Node Selector, and Tolerations](#affinity-node-selector-tolerations)
```

Comment:
You are not aligning this correctly, check with the github markdown editor and fix it.

Reply:
thanks, done